src/containers/ClassifierContainer.jsx

```diff
@@ -114,16 +114,40 @@ class ClassifierContainer extends React.Component {
               <span>Reset Image</span>
             </button>
 
-            {Object.keys(TOOLBAR_CONTROLS).map((key, i) => {
-              return (
-                <div className="block" key={i}>
-                  <span className="classifier-toolbar__icon">
-                    <i className={`${TOOLBAR_CONTROLS[key]}`} />
-                  </span>
-                  <span>{key}</span>
-                </div>
-              );
-            })}
+            <button className="flat-button block">
+              <span className="classifier-toolbar__icon">
+                <i className="fa fa-eye" />
+              </span>
+              <span>Toggle Prev. Marks</span>
+            </button>
+
+            <button className="flat-button block" onClick={this.useContrast}>
+              <span className="classifier-toolbar__icon">
+                <i className="fa fa-adjust" />
+              </span>
+              <span>Contrast</span>
+            </button>
+
+            <button className="flat-button block" onClick={this.useResetImage}>
```

Comment:
For the buttons that don't do anything _yet_ (Favorite, Collection, SubjectInfo), the `onClick` action should be empty, not `useResetImage`; this temporary functionality doesn't add anything to our testing and is a bit confusing.